runtime/components/jacLink/include/link.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <stdint.h>
+#include <stddef.h>
+#include <freertos/FreeRTOS.h>
+#include <freertos/stream_buffer.h>
+
+namespace jac::link {
+
+struct ChannelDesc {
+    StreamBufferHandle_t sb;
+    uint8_t cid;
```

Comment:
Nedávalo by smysl zatáhnout sem RAII (konstruktor + destruktor), který managuje StreamBuffer?

Reply:
Nebránil bych se tomu, ale v tuto chvíli to považuji asi za zbytečné